GiniPayBank/Classes/Core/GiniPayBank.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  GiniPayBank.swift
+//  GiniPayBank
+//
+//  Created by Nadya Karaban on 18.02.21.
+//
+
+import Foundation
+import GiniCapture
+
+//public typealias GiniPayBankCaptureNetworkDelegate = GiniCaptureNetworkDelegate
+//
+///**
+// Delegate to inform the reveiver about the current status of the Gini Capture SDK.
+// Makes use of callbacks for handling incoming data and to control view controller presentation.
+//
+// - note: Screen API only.
+// */
+public protocol GiniPayBankAnalysisDelegate : AnalysisDelegate {}
+
+@objc public protocol GiniPayBankCaptureNetworkDelegate: GiniCaptureNetworkDelegate {}
+public protocol GiniPayBankAnalysisScreenTrackingDelegate: AnalysisScreenTrackingDelegate {}
+
+public struct GiniPayBankEvent <T: RawRepresentable> where T.RawValue == String {
+    
+    /// Type of the event.
+    public let type: T
+    
+    /// Additional information carried by the event.
+    public let info: [String : String]?
+    
+    init(type: T, info: [String : String]? = nil) {
+        self.type = type
+        self.info = info
+    }
+    
+}
+
+//
+//    /**
+//     Called when the user has taken a picture or imported a file (image or PDF) from camera roll or document explorer
+//
+//     - parameter document: `GiniCaptureDocument`
+//     - parameter networkDelegate: `GiniCaptureNetworkDelegate` used to tell the Gini Capture
+//                                   Library to upload the pages upload state
+//
+//     */
+//
+//    func didCapture(document: GiniCaptureDocument, networkDelegate: GiniPayBankCaptureNetworkDelegate)
+//
+//
+//    /**
+//     Called when the user has reviewed one or several documents.
+//     It is used to add any optional parameters, like rotationDelta, when creating the composite document.
+//
+//     - parameter documents: An array containing on or several reviewed `GiniCaptureDocument`
+//     - parameter networkDelegate: `GiniCaptureNetworkDelegate` used to tell the Gini Capture SDK that the documents
+//                                   were reviewed and can be analyzed or uploaded.
+//
+//     */
+//    func didReview(documents: [GiniCaptureDocument], networkDelegate: GiniPayBankCaptureNetworkDelegate)
+//
+//    /**
+//     Called when the user cancels capturing on the camera screen.
+//     Should be used to dismiss the presented view controller.
+//     */
+//    func didCancelCapturing()
+//
+//    /**
+//     Called when the user navigates back from the review screen to the camera potentially to
+//     retake an image. Should be used to cancel any ongoing analysis task on the image.
+//     */
+//    func didCancelReview(for document: GiniCaptureDocument)
+//
+//    /**
+//     Called when the user navigates back from the analysis screen to the review screen.
+//     It is used to cancel any ongoing analysis task on the image.
+//     */
+//    func didCancelAnalysis()
+
+/**
+ Convenience class to interact with the Gini Capture SDK.
```

Comment:
Documentation is for Capture SDK, should be for Bank SDK

Reply:
I need to clean it up.